lib/faraday/adapter/em_http.rb

```diff
@@ -136,7 +136,7 @@ def perform_request(env)
           raise Faraday::ConnectionFailed, err
         end
       rescue => err
-        if defined?(OpenSSL) && OpenSSL::SSL::SSLError === err
+        if defined?(OpenSSL) && OpenSSL::SSL::SSLError == err
```

Comment:
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/CaseEquality

We want to use `is_a?` here and all of these like it.

Reply:
Thanks for the push!

Reply:
Alas, there are more of these `===` which (wrongly) got turned into `==`. Can we make them `is_a?` as well, so that this can get merged?

Reply:
Thank you for your comments!
I amended it.